packages/patternfly-4/react-core/src/components/ApplicationLauncher/ApplicationLauncher.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown, DropdownDirection, DropdownToggle, DropdownPosition } from '../Dropdown';
+import { ThIcon } from '@patternfly/react-icons';
+
+export const propTypes = {
+  /** Additional element css classes */
+  className: PropTypes.string,
+  /** Display menu above or below dropdown toggle */
+  direction: PropTypes.oneOf(Object.keys(DropdownDirection)),
+  /** Array of DropdownItem nodes that will be rendered in the dropdown Menu list */
+  dropdownItems: PropTypes.array,
+  /** open bool */
+  isOpen: PropTypes.bool,
+  /** Display the toggle with no border or background */
+  isPlain: PropTypes.bool,
```

Comment:
Do we need prop for this? Always needs to be isPlain, no?

Reply:
Not sure ... We allow it on other icon drop downs but that's just because of the way they are implemented.  I can remove it for now and we can always add it back if a request for enhancement comes in.